cmd/openshift-install/create.go

```diff
@@ -93,7 +95,13 @@ var (
 				if err != nil {
 					return err
 				}
-				return logComplete(rootOpts.dir)
+				logrus.Info("Waiting for openshift-console route to be created...")
+				consoleURL, err := waitForConsole(context.Background(), rootOpts.dir)
```

Comment:
Instead of minting multiple `Background()` contexts in one function, I think it's more idiomatic to drop a:

```go
ctx := context.Background()
```

at the front of the function and then use `ctx` where you need a context later on (for both `destroyBootstrap` and `waitForConsole`).

Reply:
done